enhancements/storage/csi-certification.md

```diff
@@ -0,0 +1,125 @@
+---
+title: csi-certification
+authors:
+  - "@fbertina"
+reviewers:
+  - "jsafrane"
+  - "@gnufied”
+  - "@chuffman"
+approvers:
+  - "@..."
+creation-date: 2019-12-18
+last-updated: 2019-12-18
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# CSI Certification
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+We want to have a CSI certification suite that our storage vendors can use to certify their CSI drivers.
+
+## Motivation
+
+As storage vendors want to include their solutions into the OpenShift ecosystem, they will need to make their CSI drivers installable via an operator. To get support for their drivers, they need to certify them and we need to provide a way to test that the driver they provide meets the key requirements.
+
+### Goals
+
+The goal is to allow a storage vendor to run a test that validates their CSI drivers for use with OpenShift as a step in the process to certify their operators for the OperatorHub.
+
+### Non-Goals
+
+* Validate CSI driver operators, this proposal targets CSI drivers only.
+* Troubleshoot or debug CSI drivers. This is the responsibility of the CSI driver vendor.
+
+## Proposal
+
+We propose that in order to be certified, a CSI driver should meet the following requirement(s):
+
+* The driver should pass all tests contained in the `openshift/csi` test suite that is part of the `openshift-tests` utility.
```

Comment:
This seems reasonable.  If we're going to do this we should convert the env vars to arguments sooner rather than later, but env vars are ok for right now as long as the description of the suite includes info about using them (i.e. the help text of openshift-tests run should clearly indicate what users should do).